### ISSUE TITLE
What Within-sample and out-of-sample means in your paper?

### ISSUE THREAD

**Issue by zhengxuyu** [AMLab-Amsterdam/CEVAE#6]
Hi,

I just read your paper. I have two questions about the meaning of Within-sample and out-of-sample in Table 1 and Table 2. 
1) I didn't see the definition of these two words in the article, could you help me to explain? 
2) Are these two Settings reflected in your code?

Thanks!

**Comment by uri-sh** [comment]
Within-sample means inferring the counterfactual outcomes for units in the training set. Note that this is different from the ordinary training set error because even for units in the training set, we do not have their counterfactuals. 
Out-of-sample means inferring both outcomes (under T=0 and T=1) for units which are not part of the training set, i.e. the standard test set.
